### PR TITLE
Zcs-11302: fixing done for 8.8.15

### PR DESCRIPTION
**Pull request by sakshirawat0247**
If the allow_username_within_password is set as false it will give error to the user that they can't use their username in password

### CODE REVIEW

File: WebRoot/messages/ZmMsg_ms.properties [hunk 1]
```diff
@@ -4571,3 +4571,18 @@ manageContacts = Lihat dan uruskan kenalan anda
 allowAccessLabel = Dengan menerimanya, anda mengizinkan aplikasi ini untuk mengakses dan menggunakan maklumat anda mengikut terma perkhidmatan dan dasar privasi Zimbra.
 oAuthFailed = Anda tida dapat didaftar masuk dengan e-mel atau kata laluan ini.
 oAuthTwoFactorFailed = Kod sekali ini tidak sah.
+<<<<<<< HEAD
```
Comment (silentsakky) [inline WebRoot/messages/ZmMsg_ms.properties:4574]: conflict markers

File: WebRoot/public/login.jsp [hunk 2]
```diff
@@ -574,41 +574,169 @@ if (application.getInitParameter("offlineMode") != null) {
...
+                                    </c:if>
+                                    <div class="passwordWrapper">
+                                        <input id="password" tabindex="2" autocomplete="off" class="zLoginFieldInput" name="password" type="password" value="" size="40" maxlength="${domainInfo.webClientMaxInputBufferLength}"/>
+                                        <span toggle="#password" onClick="showPassword();" id="showSpan" style="display: block;"><fmt:message key="show"/></span>
+                                        <span toggle="#password" onClick="showPassword();" id="hideSpan" style="display: none;"><fmt:message key="hide"/></span>
+                                    </div>
+                                    <c:set var="zimbraPasswordMinLength" value="0" />
+                                    <c:set var="zimbraPasswordMinUpperCaseChars" value="0"/>
+                                    <c:set var="zimbraPasswordMinLowerCaseChars" value="0"/>
```
Comment (silentsakky) [inline WebRoot/public/login.jsp:590]: these features don't exist in 8815

File: WebRoot/public/login.jsp [hunk 3]
```diff
@@ -108,11 +108,14 @@
 		<c:when test="${param.loginOp eq 'logout'}">
 			<zm:getDomainInfo var="domainInfo" by="virtualHostname" value="${zm:getServerName(pageContext)}"/>
 			<c:set var="logoutRedirectUrl" value="${domainInfo.attrs.zimbraWebClientLogoutURL}" />
+			<c:set var="skipLogoff" value="${domainInfo.attrs.zimbraWebClientSkipLogoff}" />
```
Comment (sakshirawat0247) [inline WebRoot/public/login.jsp:111]: These changes are not from ZCS-11302, not sure from where they are coming, should I remove these lines as well?

Comment (silentsakky) [reply WebRoot/public/login.jsp:111]: rebase issue, changes are from 8815p32

Comment (sakshirawat0247) [reply WebRoot/public/login.jsp]: done

**Review by silentsakky — APPROVED** [review]

**Pull request merged**